chart/templates/_helpers.tpl

```diff
@@ -22,20 +22,92 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+
+{{/*
```

Comment:
I don't understand all this part... I trust you

Reply:
Yep, same formatting for all our chart.